managers/tests/test_views.py

```diff
@@ -342,3 +345,104 @@ def test_remove_task_activities_view_should_change_task_status_to_not_active(sel
         self.client.post(self.url)
 
         self.assertIn(self.active_task_activity, TaskActivityType.objects.filter(is_active=False))
+
+
+class ExpectedHoursBaseTests(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.current_time = timezone.now()
+        self.user = UserFactory(
+            email="testuser@codepoets.it",
+            first_name="John",
+            last_name="Doe",
+            user_type=CustomUser.UserType.ADMIN.name,
+            is_active=True,
+        )
+
+        self.project = ProjectFactory()
+        self.client.force_login(self.user)
+        self.project.members.add(self.user)
+
+
+class ProjectExpectedHoursViewTests(ExpectedHoursBaseTests):
```

Comment:
I didn't focus on previous PR how you were naming test classes but we always should name them with starting `Test` word -> `TestProjectExpectedHoursView`

Reply:
changed